### PR TITLE
feat: new login setup

### PR DESCRIPTION
**Pull request by Akirtovskis**
Closes : https://github.com/sigle/sigle/issues/365

If there is some other place outside of screenshot where I can see exactly which properties
of styles I had to use, I am happy to revisit and update to exact values. ( talking about $mt and $gray here ) 

This is how it will look after merging this and FF on!

Screenshot 2022-03-07 at 08 26 31

### CODE REVIEW

**Comment by pradel** [comment]
For the correct margin and color everything is on figma "product design" => login!

**Comment by Akirtovskis** [comment]
Screenshot 2022-03-07 at 14 31 29

@QuentinSaubadu how does it look? 👀

File: sigle/src/pages/login.tsx [hunk 1]
```diff
@@ -75,35 +76,77 @@ const Login = () => {
...
+              {isExperimentalHiroWalletEnabled
+                ? '(*Monetisation coming soon)'
+                : `Sigle is a web 3.0 open source blogging platform focused on`}
+              {!isExperimentalHiroWalletEnabled && (
+                <span>
+                  {' '}
+                  <strong>protecting your privacy</strong>, built on top of
+                  Stacks.`
+                </span>
+              )}
```
[the remark this answers is not in the document]
Comment (Akirtovskis) [reply sigle/src/pages/login.tsx]: Done!

**Comment by Akirtovskis** [comment]
🤦 fixed!

**Pull request merged**